test/DotNetty.Transport.Libuv.Tests/EventLoopTests.cs

```diff
@@ -82,7 +82,7 @@ public void ShutdownAfterExecute()
             Assert.Throws<RejectedExecutionException>(() => this.eventLoop.Execute(this.noOp));
         }
 
-        [Fact]
+        [Fact(Skip = "flaky")]
```

Comment:
Do we have an issue/bug to track this?

Reply:
[Bug 29658901](https://msazure.visualstudio.com/One/_workitems/edit/29658901): Fix flaky test/DotNetty.Transport.Libuv.Tests/EventLoopTests.ScheduleTask